src/proxyvisor.coffee

```diff
@@ -182,17 +177,18 @@ router.put '/v1/devices/:uuid', (req, res) ->
 		console.error("Error on #{req.method} #{url.parse(req.url).pathname}", err, err.stack)
 		res.status(503).send(err?.message or err or 'Unknown error')
 
-tarPath = ({ commit }) ->
-	return '/tmp/' + commit + '.tar'
+tarPath = ({ appId, commit }) ->
+	return '/tmp/' + appId + '-' + commit + '.tar'
```

Comment:
This should be `/var/tmp` otherwise you're risking us properly mounting `/tmp` as tmpfs in the future and then storing everything in RAM